packages/pinia/CHANGELOG.md

```
@@ -8,102 +8,48 @@
-### Bug Fixes
-
-- avoid including devtools code in builds ([d3b24a3](https://github.com/vuejs/pinia/commit/d3b24a3d6a4b5af82c8ef7e66e4cecd890c30fdd)), closes [#2910](https://github.com/vuejs/pinia/issues/2910)
-
-## [3.0.0](https://github.com/vuejs/pinia/compare/pinia@2.2.6...pinia@3.0.0) (2025-02-11)
-
-This version of Pinia has no new features, it drops support for Vue 2 and other deprecated APIs. It should be an straightforward upgrade for most users! 🎉
+This version of Pinia has no new features, it drops support for Vue 2 and other deprecated APIs. It should be a straightforward upgrade for most users! 🎉
 
 See the [migration guide](https://pinia.vuejs.org/cookbook/migration-v2-v3.html) for help.
 
 ### ⚠ BREAKING CHANGES
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)